open-feature/js-sdk-contrib
# feat: Create a Growthbook server side provider

## This PR


- adds a new OpenFeature GrowthBook server provider

### How to test
Integrate OpenFeature into a test node server and set it up with this new GrowthBook provider. Ensure that GrowthBook flag evaluation works as expected using the OpenFeature API

[How to set up OpenFeature with node](https://openfeature.dev/docs/reference/technologies/server/javascript/)

OpenFeature.setProvider(new GrowthbookProvider(gbOptions, initOptions))
